sinker/backend/pktvisor/pktvisor.go

```diff
@@ -225,10 +226,12 @@ func makePromParticle(ctxt *context, label string, k string, v interface{}, tsLi
 		handleParticleError(ctxt, err)
 		return tsList
 	}
-	*tsList = append(*tsList, prometheus.TimeSeries{
+	timeSeries := prometheus.TimeSeries{
 		Labels:    labelsListFlag,
 		Datapoint: prometheus.Datapoint(dpFlag),
-	})
+	}
+	ctxt.logger.Debug("debugging timeSeries built", zap.Any("timeSeries", timeSeries))
```

Comment:
the problem is we currently run 'debug' level in production, and this would be way too noisy. so we either have to commit to not running debug level there, or remove this